.gitignore

```diff
@@ -25,3 +25,4 @@ media/
 
 # top-level .env file
 /.env
+/.venvs
```

Comment:
If you use `make devenv` to initialize the virtualenv, it will be stored in a subdirectory of `.state/`, which is already ignored by Git.